app/src/main/java/fi/metropolia/movesense/bluetooth/MovesenseConnector.kt

```diff
@@ -0,0 +1,33 @@
+package fi.metropolia.movesense.bluetooth
+
+import android.bluetooth.BluetoothManager
+import android.content.Context
+import com.movesense.mds.Mds;
+import com.movesense.mds.MdsConnectionListener
+import com.movesense.mds.MdsException
+
+class MovesenseConnector(context: Context) : MdsConnectionListener {
+    //val mds: Mds = Mds.builder().build(context)
+    private val bluetoothManager =
+        context.getSystemService(Context.BLUETOOTH_SERVICE) as BluetoothManager
+    private val bluetoothAdapter = bluetoothManager.adapter
```

Comment:
Näitä ei tarvita. Käsittääkseni Mds hoitaa kaiken Bluetoothiin liittyvän.

Reply:
Poistetaan.